README.md

```diff
@@ -55,13 +57,165 @@ And in your modules, apply the `android-apt` plugin and add the remixer dependen
 apply plugin: 'android-apt'
 
 dependencies {
-    compile 'com.github.material-foundation:material-remixer-android:develop-SNAPSHOT'
+    compile 'com.github.material-foundation.material-remixer-android:remixer_core:develop-SNAPSHOT'
+    compile 'com.github.material-foundation.material-remixer-android:remixer_ui:develop-SNAPSHOT'
+    compile 'com.github.material-foundation.material-remixer-android:remixer_storage:develop-SNAPSHOT'
     provided 'com.github.material-foundation.material-remixer-android:remixer_annotation:develop-SNAPSHOT'
 }
 ```
 
 Notice the dependency on `remixer_annotation` is a `provided` clause instead of `compile`, this is on purpose as this is not a regular dependency but a compiler plugin.
 
+### Global remixer set up
+If you have not subclassed the application class it is recommended you do it since this is a one-time global initialization.
+
+In your application class you need to add RemixerActivityLifecycleCallbacks as an ActivityLifecycleCallbacks instance, so Remixer knows to remove old variables and triggers when activities are destroyed, to avoid leaks.
```

Comment:
Or, have a "how to get started section at the top of the README, with the quickest instructions on how to integrate remixer". Then have a more detailed section at the bottom of the README.